python-client/tests/test_data/always_wrong.csv

```diff
@@ -0,0 +1,3 @@
+input,sex,output
```

Comment:
with the comment above in place, you won't need a new dataset, you'll be able to reuse the standard german credit scoring one